lib/windows/handle.rb

```diff
@@ -9,7 +9,7 @@ module Handle
 
     private
 
-    INVALID_HANDLE_VALUE           = 0xFFFFFFFF
+    INVALID_HANDLE_VALUE           = (1<<['a'].pack('p').length*8)-1
```

Comment:
I don't think that actually works. I always get 0xffffffff, regardless of whether i'm using 32-bit or 64-bit Ruby. Have you tested?